starter-code/src/viking.js

```diff
@@ -1,11 +1,38 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+  this.health = health;
+  this.strength = strength;
+  }
+  attack(){
+     return this.strength
+  }
+  receiveDamage(damage) {
+    this.health = this.health-damage;
+  }
+}
 
-// Viking
-class Viking {}
+// Vik
+
+class Viking extends Soldier {
+  constructor (health, strength, name){
+    super(health, strength)
+
+    this.name = name;
+  }
+}
+
+if this.health >= 1 {
```

Comment:
Il faudrait que tu précises que ces éléments dépendent de la function receiveDamage 
 receiveDamage(damage){
    this.health -= damage;
    if (this.health>0){
      return (this.name+" has received " + damage.toString() + " points of damage");
    } else {
    return (this.name + " has died in act of combat")
   }
  }